modules/ingest-user-agent/src/main/java/org/elasticsearch/ingest/useragent/UserAgentParser.java

```diff
@@ -26,11 +26,26 @@
 final class UserAgentParser {
 
     private final UserAgentCache cache;
+
+    private final DeviceTypeParser deviceTypeParser = new DeviceTypeParser();
+
```

Comment:
Unnecessary newlines:
```suggestion
    private final DeviceTypeParser deviceTypeParser = new DeviceTypeParser();
```